src/Widgets/VideoWidget/VideoWidgetComponent.tsx

```diff
@@ -22,7 +22,7 @@ provideComponent(VideoWidget, ({ widget }) => {
   return (
     <ContentTag
       tag="video"
-      key={`${src}${posterUrl}`}
+      key={`${src || ''}${posterUrl || ''}`}
```

Comment:
I think the code would be improved if `urlFromBinaryObj` always returns a string - an empty string if no URL could be generated.